tests/unit/config_tests.py

```diff
@@ -20,6 +20,8 @@ class TestConfigParser(unittest.TestCase):
         ('--keep-builddir', 'keep_builddir', True),
         ('--build-versions-repository-url=foo', 'build_versions_repository_url', 'foo'),
         ('--build-version=foo', 'build_version', 'foo'),
+        ('--mock-args=--enable-plugin=tmpfs --plugin-option=tmpfs:keep_mounted=True', 'mock_args',
```

Comment:
Let me explain my rationale here.

I have thought in `mock_args` as a list to keep `config.yaml` more readable so humans could clearly see parameters passed to `mock_args`, one per line:

```
mock_args:
   - --enable-plugin=tmpfs
   - --plugin-option=tmpfs:keep_mounted=True
   - --plugin-option=tmpfs:max_fs_size=50g
   - --plugin-option=tmpfs:required_ram_mb=4096
```

Otherwise, it would have to be a very long line in `config.yaml`, damaging readability:

```mock_args: "--enable-plugin=tmpfs --plugin-option=tmpfs:keep_mounted=True --plugin-option=tmpfs:max_fs_size=50g --plugin-option=tmpfs:required_ram_mb=4096"```

From what I read, you seem to prefer `mock_args` as a simple string instead of a list.  I will change that.

I will also use metasyntactic names in the unit tests.

I did appreciate your input.  Thank you!